### ISSUE TITLE
Feature request: Ctrl+Arrow keys

### ISSUE THREAD

**Issue by poweln** [hirokiky/pilemd#25]
I love this software.  However, the one thing that bothers me when entering a note is that I can't use Ctrl+"Left arrow" or Ctrl+"Right arrow" to navigate.  In most text editors, holding the Ctrl key while pressing the left or right arrow moves the cursor one word at a time.  I use that feature all the time and really miss it in PileMD.

**Comment by MattNer0** [comment]
I can't test this on Mac but the shortcut works for me on both Windows and Linux. Did you build the project yourself? Perhaps you can try my fork and see if it works there.

**Comment by poweln** [comment]
I installed originally by unzipping the Linux 0.9.0 release from pilemd.com and just running the binary.

I tried installing the 0.11 zip from your fork, and the feature works there, so thank you.  Just FYI, though, it wasn't instantly obvious to me that I couldn't create a note until I had created at least one folder.  You might consider initializing with some sort of default folder so people can just launch into creating notes.  I *love* the saving-notes-as-md-files feature though.

Thanks for your help!
